live/provision-stacker.yaml

```diff
@@ -0,0 +1,58 @@
+provision-base:
+  build_only: true
+  from:
+    type: docker
+    url: "docker://zothub.io/machine/bootkit/rootfs:0.0.5.230327-squashfs"
```

Comment:
Probably should use a substitute ${{ROOTFS_VERSION}}

Reply:
Perhaps - I just copied this from the existing stacker.yaml.